incident.py

```diff
@@ -5,10 +5,9 @@
 @task
 @roles('class-cache')
 def fail_to_mirror():
-    """Fails the site to the mirror"""
+    """Fails the site to the mirror by stopping nginx on the cache nodes"""
     puppet.disable("Fabric fail_to_mirror task invoked")
-    nginx.disable_vhost("www.gov.uk")
-    nginx.force_restart()
+    nginx_kill()
```

Comment:
Good spot. I'd cargo-culted it from https://github.com/alphagov/fabric-scripts/commit/338b44913bed4c223603dbc3b27ef0d76e2ef6d8, where it was imported explicitly as `nginx_kill`.